tests/unit/lxd/test_launcher.py

```diff
@@ -26,12 +26,12 @@
 from freezegun import freeze_time
 from logassert import Exact  # type: ignore
 
-from craft_providers import Base, ProviderError, bases, lxd
+from craft_providers import ProviderError, bases, lxd
 
 
 @pytest.fixture
 def mock_base_configuration():
-    mock_base = Mock(spec=Base)
```

Comment:
I'm hesitant about this because it could result in us missing changes we need to make to other bases. What's the reasoning behind it, and is there a way we can still spec it on Base instead (or parametrize the fixture to use each concrete base)?

Reply:
I did this because `_setup_hostname()` method isn't in the parent `Base` class.

Note - this is on `1.10` where the only base is `buildd`.

Because of Sheng's base refactoring in `1.14`, I don't have to do this "hack" on `main` (meaning I won't bring in this change when I merge this hotfix branch into `main`).

Reply:
Ahh, got it. I'm good with that then :-)